docs/eventing/getting-started.md

```diff
@@ -0,0 +1,438 @@
+---
+title: "Getting Started with Eventing"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 9
+type: "docs"
+---
+
+This getting started guide will walk you through all the steps involved in a typical Knative workflow. You will learn how to create, send, and verify events in Knative.
+
+## Before you begin
+
+You need:
+
+- A Kubernetes cluster with [Knative Eventing installed](./install/index.html). 
+
+## Setting up Eventing Subcomponents
+
+Before you start to send events, you need to create the components needed to move the events. In this guide, you will be creating components individually, but you can also create components by deploying a [single YAML file](https://raw.githubusercontent.com/akashrv/docs/qs/docs/eventing/samples/hello-world/quick-start.yaml).
```

Comment:
I'm not sure we should give folks the option of creating everything all at once. What if we re-wrote the last sentence to say, "This guide walks you through creating each Eventing subcomponent."